client/file-manager/src/traits.rs

```diff
@@ -134,7 +134,7 @@ pub trait FileStorage<T: TrieLayout>: 'static {
 
     // TODO: Return Result<Option> instead of Result only
```

Comment:
Shouldn't this be removed?

Reply:
Yep, already implemented the change so that TODO is resolved. I'll delete it